flux-driver/src/callbacks.rs

```diff
@@ -171,7 +171,7 @@ impl<'a, 'genv, 'tcx> CrateChecker<'a, 'genv, 'tcx> {
             .enums
             .into_iter()
             .try_for_each_exhaust(|(def_id, enum_def)| {
-                let enum_def = desugar::desugar_enum_def(genv, &adt_sorts, enum_def)?;
+                let enum_def = desugar::desugar_enum_def(genv, &adt_sorts, def_id, enum_def)?;
```

Comment:
I think the EnumDef has the LocalDefId as a field so you don’t need to pass it as an extra argument here.